src/lib.rs

```diff
@@ -70,19 +69,15 @@ pub fn Icon(
     svg = svg.attr(
         "width",
         leptos::Attribute::String(match (width, icon.width) {
-            (Some(a), Some(_b)) => leptos::Oco::from(a.get()),
-            (Some(a), None) => leptos::Oco::from(a.get()),
-            (None, Some(_b)) => leptos::Oco::from("1em"),
-            (None, None) => leptos::Oco::from("1em"),
+            (Some(a), _b) => leptos::Oco::from(a.get()),
+            _ => leptos::Oco::from("1em"),
         }),
     );
     svg = svg.attr(
         "height",
         leptos::Attribute::String(match (height, icon.height) {
-            (Some(a), Some(_b)) => leptos::Oco::from(a.get()),
-            (Some(a), None) => leptos::Oco::from(a.get()),
-            (None, Some(_b)) => leptos::Oco::from("1em"),
-            (None, None) => leptos::Oco::from("1em"),
+            (Some(a), _b) => leptos::Oco::from(a.get()),
+            _ => leptos::Oco::from("1em"),
```

Comment:
If `icon.width` and `icon.height` are always going to be ignored, why include them in the match expressions in the first place?